recipes/discasm/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.2" %}
+
+package:
+    name: discasm
+    version: {{ version }}
+
+source:
+  fn: DISCASM-v{{ version }}.tar.gz # [unix]
+  url: https://github.com/DISCASM/DISCASM/releases/download/DISCASM-v{{ version }}/DISCASM-v{{ version }}.tar.gz # [unix]
+  md5: 571b2265018608298752f40f3f375b0f # [unix]
+
+build:
+    number: 0
+    skip: true  # [py3k]
+    skip: True  # [osx]
```

Comment:
If you want to skip both you need to add something like this [osx or py3k]

Reply:
I'm wondering if the py4k skip is really needed.

Reply:
The DISCASM code has not been tested on python 3. It works on python 2.7.

Reply:
Regarding the python 3k skip, if skipping is not the correct way to do that, please let me know. When I recently added a recipe for slncky, I had initially just set a run requirement to be python < 3, and was told that the correct way to do that was to skip the python 3k build. Thanks for any advice you may have about the correct way to do this.

Reply:
Going ahead and pushing new version with skip [osx or py3k]. Feel free to change it to what ever it should be, if that is not right.

Reply:
Both statements are correct:
- DISCASM needs to skip builds for Python 3 since it uses Python 2-specific code, e.g. `print`-statement.
- In most (not all) cases `python` should not be pinned (see: https://github.com/conda/conda-build/issues/1962) which is also true for DISCASM, so the `skip: True` + selector is correct.